include/bluetooth/services/fast_pair.h

```diff
@@ -42,19 +57,21 @@ size_t bt_fast_pair_adv_data_size(bool fp_discoverable);
  * @param[out] adv_data		Pointer to the Bluetooth advertising data structure to be filled.
  * @param[out] buf		Pointer to the buffer used to store Fast Pair advertising data.
  * @param[in]  buf_size		Size of the buffer used to store Fast Pair advertising data.
- * @param[in]  fp_discoverable	Boolean indicating if device is Fast Pair discoverable.
+ * @param[in]  fp_adv_mode	Fast Pair advertising mode. According to Fast Pair specification,
```

Comment:
Not sure if we need to mention the implementation detail here.

Reply:
It wanted to state it somewhere. If we advertise in not discoverable mode and have no account keys we actually advertise neither in show_ui_ind mode nor hide_ui_in mode. Maybe You have an idea how to state it better?

Reply:
If you want to state it explicitly then let's leave it as it is.